pkg/virt-config/feature-gates.go

```diff
@@ -118,6 +119,14 @@ func (config *ClusterConfig) IsFeatureGateDeprecated(featureGate string) bool {
 	return false
 }
 
+var discontinuedFeaturesMessages = map[string]string{
```

Comment:
The current flow set the feature-gates  to always enabled, in Passt case it is sentimentally wrong because we  remove it form the core and provide alternative network binding plugin.
Sure its easier, less footprint but.. wrong.
From offline discussions I recall @EdDev had other concerns using the mentioned mechanism.

Also,  the mentioned flow doesn't enable customizing the meesage, its fixed one:
`feature gate ... is deprecated, therefore it can be safely removed and is redundant.`

Reply:
IMO we should extend in a backward compatible way the current mechanism. Having two is bad.

As of now, that means:
- Add an optional unique message or generalize it better.
- Provide means to control the FG check response (the boolean value).

Reply:
I agree with Eddy. The existing mechanism should be extended.